### PR TITLE
Pin News documentation V1

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
 
+Pinning a News in the home page
+-------------------------------
+
+If you have the membership "publisher" in the platform users group, you are allowed to pin articles in home page. This action is allowed from three locations.
+
+
+1- Pinning a News from creation form : 
+
+From simple and complete creation forms, a checkbox "Pin article" is available.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
From simple and full creation forms, a  "Pin article" checkbox is available. 
```

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+If you have the membership "publisher" in the platform users group, you are allowed to pin articles in home page. This action is allowed from three locations.
+
+
+1- Pinning a News from creation form : 
+
+From simple and complete creation forms, a checkbox "Pin article" is available. 
+
+|image168|
+
+When you type in fields with the article details, tick "Pin article" checkbox than click on "post" button a confirmation message appears.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
When you type in fields with the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+
+|image169|
+
+By confirming the action the article is posted to space's activity stream and automatically pinned to home page News bloc.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After confirmation,  the article is posted to space's activity stream and automatically published to home page News bloc. 
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+|image168|
+
+When you type in fields with the article details, tick "Pin article" checkbox than click on "post" button a confirmation message appears.
+
+|image169|
+
+By confirming the action the article is posted to space's activity stream and automatically pinned to home page News bloc. 
+
+
+2- Pinning a News from activity stream : the "pin article" option is available from the three dots of the posted article activity.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
2- Pinning a News from activity stream : the "pin article" option is available from the three-dots menu of the article’s activity.
```

File: docs/GettingStarted.rst [hunk 5]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+
+|image170|
+
+When you choose this option, a confirmation popup appears.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
When you choose this option, a confirmation popup appears:
```

File: docs/GettingStarted.rst [hunk 6]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+
+|image169|
+
+A success message appears after confirming the action:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After confirmation, a success message appears:
```

File: docs/GettingStarted.rst [hunk 7]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
 |image167|
 
 
+Pinning a News in the home page
+-------------------------------
+
+If you have the membership "publisher" in the platform users group, you are allowed to pin articles in home page. This action is allowed from three locations.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
As a platform-wide publisher (publisher:/platform/users role required) If you have the membership "publisher", you are allowed to pin any articles to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```

**Review by plamarque — CHANGES_REQUESTED** [review]
typos and language

File: docs/GettingStarted.rst [hunk 8]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
 |image167|
 
 
+Pinning a News in the home page
+-------------------------------
+
+As a platform-wide publisher (publisher:/platform/users role required) If you have the membership "publisher", you are allowed to pin any articles to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
As a platform-wide publisher (publisher:/platform/users role required), you are allowed to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```

File: docs/GettingStarted.rst [hunk 9]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+
+
+1- Pinning a News from creation form : 
+
+From simple and full creation forms, a  "Pin article" checkbox is available. 
+
+|image168|
+
+When you type in fields with the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After filling in the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.
```

File: docs/GettingStarted.rst [hunk 10]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+
+|image169|
+
+After confirmation,  the article is posted to space's activity stream and automatically published to home page News bloc.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After confirmation, the article is posted to the space's activity stream and automatically published to the home page's News block. 
```

File: docs/GettingStarted.rst [hunk 11]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+|image168|
+
+When you type in fields with the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.
+
+|image169|
+
+After confirmation,  the article is posted to space's activity stream and automatically published to home page News bloc. 
+
+
+2- Pinning a News from activity stream : the "pin article" option is available from the three-dots menu of the article’s activity.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
2- Pinning a News from the activity stream : the "pin article" function is also available from the three-dots menu of the article’s activity.
```

File: docs/GettingStarted.rst [hunk 12]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+
+|image170|
+
+When you choose this option, a confirmation popup appears:
```
Comment (plamarque) [inline docs/GettingStarted.rst:1270]:
```suggestion
When you select this entry, a confirmation popup appears:
```

File: docs/GettingStarted.rst [hunk 13]
```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
...
+
+After confirmation, a success message appears:
+
+|image171|
+
+
+*3- Pinning a News from News details*
+
+
+You may display the home page to verify that the news is available in the appropriate bloc.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
You may display the home page to verify that the news is available in the appropriate block.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
more language suggestions

File: docs/GettingStarted.rst (hunk 12 above)
Comment (plamarque) [reply docs/GettingStarted.rst:1270]: @samehaidi  did you deliberately decide to not accept this suggestion or did you just miss it ?

Comment (samehaidi) [reply docs/GettingStarted.rst:1270]: I decided to not accept this suggestion :)

**Review by plamarque — APPROVED** [review]
LGTM

File: docs/GettingStarted.rst (hunk 12 above)
Comment (plamarque) [reply docs/GettingStarted.rst:1270]: Fine.

**Pull request merged**